### ISSUE TITLE
done

### ISSUE THREAD

**Issue by YconquestY** [qingsenyangit/AHDRNet#11]
Feature maps should be concatenated along the channel dimension. I assume the `shape` of an input tensor is `channel × height × width`, so `dim` parameter of `torch.cat(…)` should be 0. However, `dim` is set as 1 in source codes. Can you please explain this？
